### DISCUSSION TITLE
Can the file be made into an img or iso image so that it can be guided by tools such as ventoy or GrubFM?

### DISCUSSION THREAD

**Discussion by huangsijun17** [DavidS95/Smokeless_UMAF#33] [Q&A]
I hope that tools such as [ventoy](https://github.com/ventoy/Ventoy) or [GrubFM](https://github.com/a1ive/grub2-filemanager) can be used to directly boot this program without releasing it to the root directory of the hard disk.

**Comment by Espionage724** [comment]
I wonder if the tools themselves could handle it? Theoretically just format a `FAT32` partition (tools can present GUI/flags for block device, size, etc) and copy the UMAF files to it (extract from archive)
